usr/src/cmd/sgs/lex/common/ncform

```diff
@@ -44,7 +44,7 @@ yylook(void)
 # ifdef LEXDEBUG
 	int debug;
 # endif
-	signed char *yylastch;
+	char *yylastch;
```

Comment:
I don't understand why this became (potentially) unsigned again. But I also remember that lex's signed v. unsigned issues were, at best, pretty opaque, so it's probably fine.